### DISCUSSION TITLE
Increasing the maximum temperature above 40° Celcius?

### DISCUSSION THREAD

**Discussion by joelfak** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#812] [Q&A]
Hi everyone! I have recently setup a DIY kit from https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/discussions/615 to work and got it running and connected to Home Assistant. It works perfectly and was easy to get started!

We are paying spot price for the electricy, and since the prices differ a lot during the day, I want to minimize the heating during the day by heating it up as much as possible during the night. I have added som extra insulation which works great, but it drops a few degrees to much during the day and I need to reheat it to use it in the evening.

I am thinking of increasing the maximum heat to something in the range of 43-45° Celcius (109-113° F), in order for it to be at a good temperature when it is time to use it. Has anyone experimented with this? I know the LayZ pumps are restricted to 40° C, but I am thinking that the restriction is located in the display unit. Does anyone know anything about this subject, or has a good reason to why I should not? Otherwise I think I will look into the code and start to experiment a bit.

**Comment by visualapproach** [comment]
Hi. Not possible on 6-wire pumps since the device emulates someone pushing the buttons on the display. On the 4-wires it will trigger the overheat protection in the pump.

I advise against raising the temperature above 40C. Someone could get burnt accidentally, and the materials in the pump could potentially break and cause bad things to happen.

**Comment by joelfak** [comment]
Hi, thanks for your answer. I suspected that it was not possible (I have the 6-wire type).

I have had those concerns as well, so I hadn't made up my mind yet. The plan was to only increase the temperature only a few degrees to mitigate this risk and only run it by night, long before anyone would use it. But things can go wrong, bugs etc. And since it is impossible I don't need to think more about it. 

I guess the only way to go is more insulation then, I have ordered a thermal cover which will help a bit more I hope.

Thank you for this project! It has been fun setting it up and really useful! I will send you some coffee :)

**Reply by visualapproach** [reply]
Yes, maybe having a bit more water in the tub would take longer to cool. But insulation is probably the best solution.
Glad to hear you enjoyed the project! Thank you 😊
